.gitmodules

```diff
@@ -1,9 +1,9 @@
-[submodule "externals/mdspan"]
-	path = externals/mdspan
-	url = https://github.com/kokkos/mdspan
 [submodule "externals/magic_enum"]
```

Comment:
Thoughts for later: We aren't using `magic_enum` in our apps so far as it requires advanced compiler features not available in `nvc++` yet. We can remove this and it's instances if any from CMake files too to avoid baggage.